src/core/lv_obj.h

```diff
@@ -115,7 +115,11 @@ enum {
     LV_OBJ_FLAG_USER_2          = (1L << 28), /**< Custom flag, free to use by user*/
     LV_OBJ_FLAG_USER_3          = (1L << 29), /**< Custom flag, free to use by user*/
     LV_OBJ_FLAG_USER_4          = (1L << 30), /**< Custom flag, free to use by user*/
+    LV_OBJ_FLAG_SCROLL_CHAIN_VER = (1L << 31), /**< Allow propagating the vertical scroll to a parent*/
```

Comment:
Please move it after `LV_OBJ_FLAG_SCROLL_CHAIN_HOR` and increment the values below it by one.